### ISSUE TITLE
[Inquiry] Mass update the repo images?

### ISSUE THREAD

**Issue by pswiifanwmi** [maxnet/berryboot#348]
## I'm sure you get this question **a lot**, but do you yourself host the images on Berryboot?  If so, is it possible for you to update all of them to the latest version?  **Please?**  I made a nice handy dandy list of updates with some nice handy dandy links.

~~-Ubuntu MATE 15.04 to [16.04](https://ubuntu-mate.org/raspberry-pi/)~~
-[Snappy Ubuntu Core?](https://developer.ubuntu.com/en/snappy/start/raspberry-pi-2/) to replace/update Ubuntu Core
-Fedora 21 MATE to [Fedora 25 MATE-Compiz](https://arm.fedoraproject.org/) (and note for Raspberry Pi 2 and 3 only since those are the only ones supported.)(And you can choose any of the DEs if MATE-Compiz is too slow or big)
-[Pingus ](https://pignus.computer/)for the Pi 0 and 1 to replace Fedora since Fedora dropped support
-[RTAndroid](https://git.embedded.rwth-aachen.de/rtandroid/downloads/raspberry-pi/) for Pi 3 ONLY
~~-LibreELEC 7.0.1 to [7.0.2](https://libreelec.tv/downloads/)~~
-Add [OSMC](https://osmc.tv/download/)?
-Kali Linux 2.0.1 Lite to [2.1.2 on this page](https://www.offensive-security.com/kali-linux-arm-images/)
-Update ArchLinux ARM to whatever the latest is now [Pi 2 and Pi 1](https://archlinuxarm.org/about/downloads)
-Update that Puppy Alpha to the NEW [Puppy Linux Quirky Xerus](http://puppylinux.org/main/Download%20Latest%20Release.htm)!
-Update RetroPie 3.6 to [4.1](https://retropie.org.uk/download/)
-Add [Lakka](http://www.lakka.tv/)?  It's on NOOBS.
~~-Update Raspbian Jessie Lite to [2016.09](https://www.raspberrypi.org/downloads/raspbian/)~~
-Update Sugar to [Fedora 25](https://arm.fedoraproject.org/). Fedora 14 is too old
-OpenMediaVault from 2.2.1 to [3.0.51](https://sourceforge.net/projects/openmediavault/files/Raspberry%20Pi%20images/)

Wow...  As I was writing this I realized, this is a lot!  If you need help with Squashifying, I'm sure I and everyone else reading this could help using that guide you provided.  I have gotten the latest Ubuntu MATE, Kali Linux, and Lakka to work on Berryboot, so those at least can be updated as well as Fedora?  The sooner the better!

Also, I know this is a lot to ask, but can you update the logos?  Preferably for Kali, Ubuntu MATE, Puppy QX, Memtester, Sugar, and Raspbian?  I personally would really enjoy that, as would probably other users.

Again, I really appreciate the work you put into this awesome bootloader and would be grateful if you could update your repositories.

Anyone else have any suggestions to add?

Thanks a bunch!

**Comment by ZanderBrown** [comment]
They are hosted on sourceforge

**Comment by maxnet** [comment]
Yes, should all update them some day when I have time.

Do wonder if all are still supported OSes though.

E.g.
"- Ubuntu MATE 15.04 to 16.04"
For Intel architectures Ubuntu is already at 16.10
No new release for the Pi?

**Comment by pswiifanwmi** [comment]
I'm sure Ubuntu MATE will still be supported despite that fact.  I bet they intend to keep the Pi on 16.04 LTS because it's LTS, but I do wonder, because they haven't released a 16.04.1 image yet...  Guess we'll wait till January for 16.04.2 to roll around.  It's still actively updated though, just like Kali.  Kali is constantly updating their rolling releases, which is what the Pi's image is.  They just don't update the image.  The only one I'm sure is unsupported is Puppy.

Just checked Sourceforge and thanks for updating these images so quickly!  You rock!

**Comment by pswiifanwmi** [comment]
Another question, do you make the images on SourceForge using the same method as on your website, because the file extensions are different for some of them.  Is there a reason for that?

**Comment by maxnet** [comment]
The Number in the extension indicates the preferred memory split.
No number means it does not care.

**Comment by pswiifanwmi** [comment]
Huh.  And my first question?

**Comment by maxnet** [comment]
Depends on operating system.
E.g. OpenELEC's SYSTEM file is already in the right squashfs format.
So with that don't bother with kpartx and mksquashfs but just rename their SYSTEM file to OpenELEC_version.img128

**Comment by pswiifanwmi** [comment]
Interesting.  So OpenELEC comes without a kernel?  Or do I not understand the process?

**Comment by maxnet** [comment]
No, it comes with a kernel, initramfs and SYSTEM file with everything else.
Berryboot can replace the first two.

**Comment by pswiifanwmi** [comment]
So, if you were to leave the kernel files in the image, BerryBoot would just replace it?

**Comment by maxnet** [comment]
Not sure if I understand your question.
But Berryboot's kernel is always used for all images.

**Comment by pswiifanwmi** [comment]
Hmm.  Okay.  I understand.

Another question, when updating Raspbian, sometime a package labeled Raspberrypi_kernel or something installs and breaks my ability to boot.  Is there a way to fix that?

**Comment by pswiifanwmi** [comment]
Well, this man has certainly out done himself, and I for one, am happy I found this!

http://berryboot.alexgoldcheidt.com/

**Comment by pswiifanwmi** [comment]
Oh, Puppy Linux Quirky Xerus has been out for the past couple months and no one noticed...  Huh.

_Edits Original Post_
